### ISSUE TITLE
Abnormality of bronchial artery

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6525]
**Preferred term label:** Abnormality of bronchial artery

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal systemic arterial morphology HP:0011004

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777
Chronic Thromboembolic Pulmonary Hypertension ORPHA:70591
 

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Adding as follows, reopen if needed @eswietlik 
label: Abnormal bronchial artery morphology
def: Any structural abnormality of a bronchial artery. 
Comment:The bronchial arteries originate from the systemic circulation and supply the lungs with oxygenated blood.
